rascal-lsp/RELEASE-NOTES

```diff
@@ -0,0 +1 @@
+../rascal-vscode-extension/README.md
```

Comment:
that is not needed, `rascal-lsp/RELEASE-NOTES.md` already exists.

Reply:
don't we have a duplicate then? or is this a different kind of release notes?

Reply:
yes, as that document describes it's duplicate, primarily because the versions of vs code and rascal-lsp don't align. And the release steps have been updated to include writing this duplicate entries.

Reply:
for me rascal-lsp/RELEASE-NOTES.md is a new file. How did I get myself into this situation?